apiserver/facades/controller/remoterelations/remoterelations.go

```diff
@@ -430,3 +439,44 @@ func (f *RemoteRelationsAPI) SetRemoteApplicationsStatus(args params.SetStatus)
 	}
 	return result, nil
 }
+
+// UpdateControllerForModel is not available via the V1 API.
+func (u *APIv1) UpdateControllerForModel(_, _ struct{}) {}
+
+// UpdateControllersForModels changes the external controller records for the
+// associated model entities. This is used when the remote relations worker gets
+// redirected following migration of an offering model.
+func (api *API) UpdateControllersForModels(args params.UpdateControllersForModelsParams) (params.ErrorResults, error) {
```

Comment:
I think horse has bolted on always using a collection for API methods, but I honoured it here.